Assets/Scripts/Traffic System/GraphMovement.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GraphMovement : MonoBehaviour
+{
+    [SerializeField] Pathfinding path; //pathfinding
+    [SerializeField] MarkGraph map; //all of the nodes
+    public List<NodeMarker> pathway; //pathway that car follows
+    public NodeMarker node1;
+    public NodeMarker node2;
+    public float speed = 5.0f;
+
+    public int position = 0;
```

Comment:
Serialize speed, and eventually make these private after you no longer need them public for testing working with music